src/kernels/dynamic_igemm/igemm_wrw_gtc_gfx908/igemm_wrw_gtc_gfx908_004x064.inc

```diff
@@ -760,7 +760,7 @@ L_igemm_wrw_gtcx_nchw_fp32_bx1_ex1_bt4x64x16_wt4x64_ws1x1_wr1x1_ta1x1x1x1_1x16x1
     ;v_add_co_u32 v34, vcc, 0, v[v_a0+2]
     v_mov_b32 v[v_tmp], s[s_in_offset]
 
-    global_store_dword v[v_tmp+6:v_tmp+7], v[v_in_os], s[s_dbg+0:s_dbg+1]
+    buffer_store_dword v[v_in_os], v[v_tmp+6], s[s_p_wei+0:s_p_wei+3], s[s_tmp] offen
```

Comment:
@shaojiewang I guess here should be `buffer_store_dword v[tmp+6], v[v_in_os], s[s_p_wei+0:s_p_wei+3], s[s_tmp] offen`, for syntax for buffer_store should be `buffer_store_dword                 vdata,           vaddr,  srsrc,  soffset idxen offen offset12 glc slc`. All the following changes I believe you write the `vdata` and `vaddr` wrong order.

Reply:
or do you mean `v_in_os` hold the data? If so, I think it is quite miss-leading. and you'd better change to a more descriptive label, or other people maintain this piece of code will confused. Please note since we use label to mark each vgpr/sgpr index with a proper meaning, you'd better follow this guide line, it's really important to keep the syntax even you write asm

Reply:
Same as above, this code branch is to help developer to debug the asm code. This store want to check if v_in_os register holds the right result.

Reply:
OK, in this case please remove this debug code

Reply:
yes, it will be removed today.